### ISSUE TITLE
A11y: when viewing items, readonly fields lack accessible name/label

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/browser#1984]
## Describe the Bug

When viewing an item, such as a login, the various pieces of data are presented as `readonly` form controls. These controls lack an accessible name (i.e. assistive technologies will just announce them as unlabelled/generic fields like "text entry, readonly").

```
<div class="row-main">
  <span class="row-label draggable" draggable="true">Username </span>
  <input aria-readonly="true" readonly="readonly" type="text">
</div>
```

In contrast, in the edit view, the fields have an associated `<label>` that provides an appropriate/correct accessible name for the controls.

```
<div appboxrow="" class="box-content-row">
  <label for="loginUsername">Username</label>
  <input appinputverbatim="" id="loginUsername" inputmode="email" name="Login.Username" type="text" autocomplete="off" autocapitalize="none" autocorrect="none" spellcheck="false" class="ng-untouched ng-pristine ng-valid">
</div>
```

## Steps To Reproduce

1. Using a screen reader, navigate through an item view (using `Tab` to jump between the different readonly form fields)

## Expected Result

* when focus lands on a readonly form control, it announces the visible label (e.g. "Username") as the name of the readonly field

## Actual Result

* the various readonly form fields lack an accessible name and are announced as generic/unlabelled form controls (with their type and value only, no actual name/label as well)

## Screenshots or Videos

![Part of a login item view](https://user-images.githubusercontent.com/895831/127718258-a6b6ca62-f2e4-4a28-8055-41916253e2fe.png)

## Environment

- Operating system: Windows 10
- Browser: Chrome 92
- Build Version (go to "Settings" → "About" in the app): 1.51.1

## Additional Context

Angular is not my forte, but presumably the markup can be mostly cribbed from the edit view and put into the item view itself.
